### PR TITLE
Add stale issue/pr action

### PR DESCRIPTION
**Pull request by nicksellen**
#### Proposed Changes

* mark our unloved issues and PRs as unloved

#### Testing Instructions

* nothing will happen until it's merged and it runs on it's schedule

Fixes #aloadofoldissuesnobodycaresaboutanymore

### CODE REVIEW

File: .github/workflows/stale.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v2.0.0
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+
+          # See https://github.com/actions/stale/blob/master/action.yml for description of the options
+
+          stale-issue-label: unloved
```
Comment (simison) [inline .github/workflows/stale.yml:16]: We have a few labels with [Status] prefix, might work here too?

"[Status] unloved" would be quite a combo of serious and less serious, haha!

File: .github/workflows/stale.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+          stale-issue-message: |
+            This issue is marked as unloved because it has not had any activity for 180 days.
+
+            It doesn't mean it's not important, so please remove the unloved label if you like it, or add a comment saying what it means to you :)
```
Comment (simison) [inline .github/workflows/stale.yml:21]: Could add something about "if this was a big, please test to see if this is still an issue?"

File: .github/workflows/stale.yml (hunk 1 above)
Comment (nicksellen) [reply .github/workflows/stale.yml:16]: I kinda feel those `[Status]` ones are for humans to mark... _unloved_ is quite short and sweet...

Comment (simison) [reply .github/workflows/stale.yml:16]: Works for me 👍

**Pull request merged**